src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -248,6 +248,44 @@ public function createTable(Table $table, array $columns = [], array $indexes =
         }
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    protected function getChangePrimaryKeyInstructions(Table $table, $newColumns)
+    {
+        $instructions = new AlterInstructions();
+
+        // Drop the existing primary key
+        $primaryKey = $this->getPrimaryKey($table->getName());
+        if (!empty($primaryKey)) {
+            $this->dropPrimaryKey($table->getName(), $primaryKey);
```

Comment:
Can this be done through `$instructions`? It's not nice that it is being done immediately right now. The simplest way is to make `dropPrimaryKey` return another `AlterInstructions` object you can merge here. Then you can rename the function to `getDropPrimaryKeyInstructions`

Reply:
I remember it was painful to have to write it this way, but I was having some difficulty adding it to $instructions... I'll try again in this iteration since the code is simpler now too, and I'll ask here if I have trouble again.

Note to self: it's the same in SqlServerAdapeter!